src/Components/MovieForm/MovieForm.jsx

```diff
@@ -14,87 +14,158 @@ const defaultMovieValues = {
 };
 
 export const MovieForm = ({ initialValues = defaultMovieValues, onSubmit }) => {
-  const [movie, setMovie] = useState({
-    ...defaultMovieValues,
-    ...initialValues,
-  });
+  const validateForm = (values) => {
```

Comment:
Also, just FYI, in more complicated cases for validation usually something like `zod` or `yup` is used - https://formik.org/docs/guides/validation#validationschema.